examples/tictactoe/src/constants/chains.tsx

```diff
@@ -0,0 +1,19 @@
+import { Chain } from 'viem'
+
```

Comment:
these exist in our viem package https://github.com/ethereum-optimism/ecosystem/blob/main/packages/viem/src/chains/supersim.ts i think we should try to use them in this app so if we need to make it easier to use everyone can benefit to get up and running with supersim easier

Reply:
updated so that it reads from the viem package